### DISCUSSION TITLE
Is there a way to inject saves on the standalone Xbox Series X flycast?

### DISCUSSION THREAD

**Discussion by YungPrazo** [flyinghead/flycast#1385] [General]
Was searching around and couldn't find anything on this topic. Thanks for the help!

**Comment by flyinghead** [comment]
You can use Dream Explorer, which has around 6000 vmu saves.
Another option is VMU CD Dream On Edition (https://www.bucanero.com.ar/downloads/)
Both are .CDI disks.

**Reply by YungPrazo** [reply]
Hey thanks so much for the saves! But I'm just curious how to inject these saves? I can't find where the saves are stored for standalone flycast on the series X.

**Reply by MesonW** [reply]
Not sure if you used the options suggested, but in case it's helpful, the location of saves on Xbox Series is here...

User Folders \ LocalAppData \ < big long string of characters for Flycast > \ LocalState \ data \

This is accessible through the Dev Portal for your Xbox.
It contains the vmu_save_A1.bin, B1, etc files if you're using standard emulated VMUs, and < game filename >_vmu_save_A1.bin if you've set `Per Game VMU A2`

**Comment by flyinghead** [comment]
Run the CDI and copy the save you want to one of the VMUs. Then run the game.
Don't save to the A1 vmu if using the `Per Game VMU A1` option.
